cumulus/scripts/ci/changelog/README.md

```diff
@@ -0,0 +1,77 @@
+# Changelog
+
+Currently, the changelog is built locally. It will be moved to CI once labels stabilize.
```

Comment:
:sparkles: nice

Reply:
It did not go over the content...

Reply:
On the other hand, this is true again now :)